docs/about.md

```diff
@@ -27,8 +27,7 @@ suggest ideas in any of our [community areas](./community.md).
   - Pulsar is being made by a community who came together from the stellar
     remnants of Atom. A community that wants to build upon the huge legacy that
     was left and make a uniquely hackable editor.
-- To continue and build upon the legacy of the Atom text editor which is
-  being/has been sunset.
+- To continue and build upon the legacy of the Atom text editor which has been sunset.
```

Comment:
```suggestion
- To continue and build upon the legacy of the Atom text editor which has been [sunset](https://github.blog/2022-06-08-sunsetting-atom/).
```

Reply:
Seems these changes weren't addressed prior to being marked as resolved?

Reply:
I have done so @confused-Techie. Thank you.